### PR TITLE
[build, ci] Add SDK Registry support.

### PR DESCRIPTION
**Pull request by tarigo**

### CODE REVIEW

**Review by nagineni — APPROVED** [review]

File: build.gradle [hunk 1]
```diff
@@ -3,11 +3,27 @@ buildscript {
...
+                password = project.hasProperty('SDK_REGISTRY_TOKEN') ? project.property('SDK_REGISTRY_TOKEN') : System.getenv('SDK_REGISTRY_TOKEN')
+            }
+        }
     }
 
     dependencies {
         classpath pluginDependencies.gradle
         classpath pluginDependencies.mapboxSdkVersions
+        if (SDK_REGISTRY == 'true') {
+            println 'SDK_REGISTRY=true'
```
Comment (mr1sunshine) [inline build.gradle:22]: nit: do we need this log line?

File: build.gradle [hunk 2]
```diff
@@ -3,11 +3,27 @@ buildscript {
...
+            classpath pluginDependencies.mapboxSdkRegistry
+        } else {
+            println 'SDK_REGISTRY=false'
```
Comment (mr1sunshine) [inline build.gradle:25]: nit: do we need this log line?

File: package.json [hunk 3]
```diff
@@ -14,8 +14,8 @@
     "url": "git://github.com/mapbox/mapbox-events-android.git"
   },
   "devDependencies": {
-    "express": "^4.11.1",
     "@octokit/rest": "^15.15.1",
+    "express": "^4.11.1",
```
Comment (mr1sunshine) [inline package.json:18]: nit: do we need this change?

**Review by mr1sunshine — APPROVED** [review]
LGTM with nits

**Pull request merged**